yarn-project/end-to-end/src/e2e_fees/fees_test.ts

```diff
@@ -187,22 +189,29 @@ export class FeesTest {
 
         await harness.bridgeFromL1ToL2(this.INITIAL_GAS_BALANCE, this.INITIAL_GAS_BALANCE, bananaFPC.address);
 
+        const testContract = await TestContract.deploy(this.aliceWallet).send().deployed();
```

Comment:
I may have missed it, but are you using the test contract at all in the e2e fees test? Could we use the token contract for test txs instead?

Reply:
No, indeed, I was going to in order to show reverts in a buggy fee payment contract, but decided it would be better to just wait until we can simply revert due to out of gas.

I'll remove it!